isis/src/osirisrex/objs/OsirisRexTagcamsCamera/unitTest.cpp

```diff
@@ -0,0 +1,175 @@
+/** This is free and unencumbered software released into the public domain.
+
+The authors of ISIS do not claim copyright on the contents of this file.
+For more details about the LICENSE terms and the AUTHORS, you will
+find files of those names at the top level of this repository. **/
+
+/* SPDX-License-Identifier: CC0-1.0 */
+
+#include <iomanip>
+#include <iostream>
+
+#include "Camera.h"
+#include "CameraFactory.h"
+#include "IException.h"
+#include "iTime.h"
+#include "OsirisRexTagcamsCamera.h"
+#include "Preference.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+
+using namespace std;
```

Comment:
Im guessing this was written a while ago, is it possible to change this to a GTEST instead? Especially since some GTESTs were already written as part of the PR.

Reply:
Removed and replaced with gtests.